packages/x-charts/src/ChartsTooltip/ChartsTooltipContainer.tsx

```diff
@@ -106,6 +106,7 @@ function ChartsTooltipContainer(inProps: ChartsTooltipContainerProps) {
   } = props;
 
   const svgRef = useSvgRef();
+  const anchorRef = React.useRef<SVGRectElement | null>(null);
```

Comment:
I don't understand why this element is necessary. Wouldn't it be possible to just provide a fake element like `pointerAnchorEl`?